### PR TITLE
Reformatted some of the view, added a second review to the seed file

### PR DESCRIPTION
**Pull request by darksim735**
image

### CODE REVIEW

**Comment by itsaliu90** [comment]
:hamburger:, the second Review will conflict once we merge @carolynyoo's https://github.com/carolynyoo/stack_store/pull/40, but we should overwrite this change with her's.

File: seed.js [hunk 1]
```diff
@@ -140,7 +140,13 @@ var seedReviews = function () {
         comment: "This movie is cool",
         rating: 4,
         film: "5536f882712a688124e77b80"},    // reference film collection
-              
+      
+        {user: "5532c9d007456cf6297ab1f5",    // reference user collection
+        date: "Wed Jul 27 16:54:49 EST 2011",
```
Comment (carolynyoo) [inline seed.js:145]: is this a date object? looks like a string to me and im concerned why the schema didn't catch this as error when seeding

File: browser/js/pdp/productimage.html [hunk 2]
```diff
@@ -1,2 +1,4 @@
 <p>Image Location: {{film.photo}}</p>
```
Comment (carolynyoo) [inline browser/js/pdp/productimage.html:1]: what is this for?

Comment (darksim735) [inline browser/js/pdp/productimage.html:1]: That was just for debugging earlier. We can delete

**Comment by darksim735** [comment]
Redid this on a different branch without the new merge conflicts from seed

**Pull request closed without merging**